razorpay-quick-payments.php

```diff
@@ -4,7 +4,7 @@
  * Plugin Name: Razorpay Quick Payments
  * Plugin URI: https://github.com/razorpay/razorpay-quick-payments
  * Description: Quick Payments for Wordpress, by Razorpay.
- * Version: 1.2.2
+ * Version: 1.2.1
```

Comment:
@jampulanaveen Upgrade the version to 1.2.3

Reply:
ok